.github/workflows/workflow_finetune.yml

```diff
@@ -113,14 +113,14 @@ jobs:
           EOF
           )
           docker exec "finetune" python -c "$CMD"
-          docker exec "finetune" bash -c "python finetune/finetune.py --config_file finetune/finetune.yaml"
+          docker exec "finetune" bash -c "python -m llmonray.finetune.finetune --config_file llmonray/finetune/finetune.yaml"
```

Comment:
llmonray.finetune.finetune is verbose. Use module llm_on_ray.finetune, may need to expose interfaces in module `__init__.py` and run from it.
Another better way is to use setup.py to install a command into bin and call `llm_on_ray-finetune` comand line instead, you can check how to do this from rayllm.